yaeger.spacesimulator/src/main/java/com/yaeger/spacesimulator/scenes/SimulationScene.java

```diff
@@ -23,14 +30,54 @@ public void setupScene() {
 
 	@Override
 	public void setupEntities() { 
-		objects.add(new Planet(new Coordinate2D(10, 10), new Coordinate2D(1, 0), 1, 100, 10, new Color(0.70D, 0.70D, 0.70D, 1.0D)));
-		
 		objects.forEach(obj -> { this.addEntity(obj); });
 	}
 	
 	@Override
 	public void explicitUpdate(long timestamp) {
-		// TODO Auto-generated method stub
 		simulationUpdater.updateSimulation(objects);
 	}
+	
+	private void placePlanet(ObjectPlacementData data) {
+		Planet planet = new Planet(data.getStartPosition(), data.getDirection(), data.getVolume(), data.getDensity(), data.getVelocity(), data.getColor() );
+		this.objects.add(planet);
+		this.addEntity(planet);
+	}
+	
+	@Override
+	public void onMouseButtonPressed(MouseButton button, Coordinate2D coordinate2d) {
+		// TODO Auto-generated method stub
+		System.out.println(String.format("CLICKED %s - %f, %f", button.name(), coordinate2d.getX(), coordinate2d.getY()));
```

Comment:
Check even de comments en printlines die er nog in staan

Reply:
Idem, deze code staat niet op GIt. Deze comments zijn allemaal al weg.